pkg/agent/types.go

```diff
@@ -42,7 +42,7 @@ type NodeBootstrappingConfiguration struct {
 	UserAssignedIdentityClientID  string
 	ConfigGPUDriverIfNeeded       bool
 	EnableGPUDevicePluginIfNeeded bool
-	EnableDynamicKubelet          bool
```

Comment:
FYI https://github.com/Azure/AgentBaker/pull/410/files# this PR removes pkg/agent/types.go